### PR TITLE
games-simulation/openrct2: bump to 0.2.2

### PR DESCRIPTION
**Pull request by l29ah**

### CODE REVIEW

File: games-simulation/openrct2/openrct2-0.2.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (mgorny) [inline games-simulation/openrct2/openrct2-0.2.2.ebuild]: EAPI 7, please.

File: games-simulation/openrct2/openrct2-0.2.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+pkg_postinst() {
+	if [[ -z ${REPLACING_VERSIONS} ]]; then
+		ewarn ""
+		ewarn "You need the original RollerCoaster Tycoon 2 files to play this game."
+		ewarn "See: https://github.com/OpenRCT2/OpenRCT2/wiki/Required-RCT2-files#how-to-retrieve"
+		ewarn ""
+	fi
+	gnome2_icon_cache_update
```
Comment (mgorny) [inline games-simulation/openrct2/openrct2-0.2.2.ebuild]: (EAPI 7 implies switching to `xdg_icon_cache_update` here, from xdg-utils)

**Comment by mgorny** [comment]
@jansegre, @lorem-ipsum, you alive?

@l29ah, want to become the third proxied maintainer for this package?

**Comment by jansegre** [comment]
Hey, it's been a while since I haven't used my gentoo system. I can't review this promptly.

**Comment by mgorny** [comment]
You don't have to test it, I'll test it anyway. Looking at the change would be appreciated though.

**Comment by jansegre** [comment]
The version changes look good and seem to follow the upstream changes correctly. A patch was removed and also a cmake flag (`-DDISABLE_RCT2_TESTS=ON`) which may be due to the upstream build improving a little and needing less workarounds, but those changes I can't test.

**Comment by mgorny** [comment]
Ok, thanks. I'm still waiting for update on requested changes though.

**Comment by juippis** [comment]
@l29ah ping.

**Comment by klardotsh** [comment]
openrct2-0.2.4 came out a week and some change ago: https://github.com/OpenRCT2/OpenRCT2/releases/tag/v0.2.4

Would it be sensible to add that version to this PR or should that be separate? Either way - happy to test ebuilds for this game if more help is needed to get these through!

**Comment by juippis** [comment]
This PR seems dead, make a new one. Additionally, @conikost you interested in taking over? Seems like you've done latest changes to this package.

**Comment by ConiKost** [comment]
@juippis I can do, let me have a look. I will see and submit a new PR.

**Pull request closed without merging**